src/views/tutorial-view/components/next-previous/components/directional-link-box/types.ts

```diff
@@ -0,0 +1,9 @@
+import { CardLinkProps } from 'components/card-link/types'
+export type DirectionOption = 'next' | 'previous' | 'final'
```

Comment:
Two things here:

- Need a space between these lines
- `CardLinkProps` is exported from the component file, so the `/types` isn't needed in the import statement

```suggestion
import { CardLinkProps } from 'components/card-link'

export type DirectionOption = 'next' | 'previous' | 'final'
```